packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -42,7 +43,12 @@ export function Header({
     [setLayout, layout]
   );
 
-  if (isMobile || !enabledLayouts) return null;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => {
```

Comment:
I would prefer to not use useEffect. Is there a way to fix the hydration error without useEffect ?
https://react.dev/learn/you-might-not-need-an-effect

Reply:
We will try to fix it without useEffect